json struct.py

```diff
@@ -48,26 +48,26 @@ def get_header_line(key, value):
     t = get_type(value)
 
     if type(value) == bool:
-        return f"\tbool {key} = {str(value).lower()};\n"
+        return f"    bool {key} = {str(value).lower()};\n"
```

Comment:
I don't mind changing, as long as you explain your thinking here.
This is for a C++ project. 
~I've been using tabs there.~ It seems my editor has been converting my tabs.

Reply:
Yes, the existing C++ code uses spaces just about everywhere. This will make the generated code consistent with that.